File: LV1/1.3/grassseed/grassseed.go
package main

import "fmt"

func main() {
	var c float64
	var n int

	fmt.Scanf("%f", &c)
	fmt.Scanf("%d", &n)

	ans := 0.0

	for i := 0; i < n; i++ {
		var w, h float64
		fmt.Scanf("%f %f", &w, &h)
		ans += w * h
	}
	fmt.Println(ans * c)
}

File: LV1/1.3/grassseed/grassseed.cpp
#include <cstdio>

int main() {
    double c;
    int n;

    scanf("%lf", &c);
    scanf("%d", &n);

    double ans = 0;

    for (int i = 0; i < n; i++) {
        double w, h;
        scanf("%lf %lf", &w, &h);
        ans += w * h;
    }
    printf("%.6lf\n", ans * c);
}
